patchwork-capabilities/README.md

```diff
@@ -0,0 +1,27 @@
+## Reasoning
+MinecraftForge offers a capability API
+
+## TODO
```

Comment:
Should probably address at least some of this before merging

Reply:
It's not really possible

Reply:
I think the IForgeItem and IForgeDimensions should be resolved, however the special cases should probably be addressed in a separate PR. I think that the remaining TODOs (after patcher is fixed) should live in a tracking issue instead of in this README file.